Cue/app/tabs/account/AccountHome.js

```diff
@@ -64,9 +63,38 @@ type Props = {
 }
 
 class AccountHome extends React.Component {
+
   _logOut = () => {
-    LoginManager.logOut()
-    this.props.logOut()
+    if (this.props.localChanges && this.props.localChanges.length) {
+      this.props.syncLibrary(this.props.localChanges).then(failedSyncs =>{
+        if (failedSyncs && failedSyncs.length) {
+          Alert.alert(
+            'You have local changes which conflict with changes in the Cue cloud',
```

Comment:
Title Case: `You Have Local Changes Which Conflict with Changes in the Cue Cloud`